src/nationalcareers_toolkit/assets/src/frontend/sass/dysac/_results.scss

```diff
@@ -1,3 +1,9 @@
+#whatYouToldUs {
```

Comment:
@defc0nONE just to confirm are these changes meant to be in this release? DYSAC?

Reply:
Looks like this belonged to a previous feature team's work that we branched off of: https://skillsfundingagency.atlasian.net/browse/FT2-1340

I've recreated the on prod vs staging and while this _does_ technically fix the issue, this wouldn't have been part of our stream of work. Should we keep or remove?